docs/zh/data_source/feature-support-data-lake-analytics.md

```diff
@@ -172,6 +172,21 @@ Iceberg Catalog 支持 HMS、Glue 和 Tabular 作为其元数据服务。大多
 | Distributed metadata plan (推荐在元数据体量较大的场景使用)   | v3.3+    |
 | Manifest Cache (推荐在元数据体量较小但对返回时延较为敏感的场景使用) | v3.3+    |
 
+自 v3.3.0 起，StarRocks 支持上述的元数据读取和缓存策略。策略的选择会根据您集群的机器情况来自动调整，通常您不需要进行修改。由于开启了元数据缓存，有可能因性能牺牲了元数据新鲜度，您可以根据具体的查询要求进行调整：
+
+- **[默认设置 推荐] 性能最佳，可以容忍分钟级别的数据不一致**
+  - **设置**：无需设置，默认情况下，10 分钟内更新的数据不可见。在此期间，查询将返回旧数据。
+  - **优点**：查询性能最佳
+  - **缺点**：延迟引起的数据不一致
+- **数据导入（产生新文件）立即可见，不依赖手动 Refresh**
+  - **设置**：通过将 Catalog 属性 `iceberg_meta_cache_ttl_sec` 设置为 `0` 禁用 db/partition/datafiles/deletefiles 的元数据缓存。
```

Comment:
we need to set `iceberg_table_cache_ttl_sec` to `0`, because ingesting new data also generates a new snapshot.